spec/helpers/shopping_lists_helper_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe ShoppingListHelper, type: :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
- [ ] There is an error stopping your test from running, kindly fix it
![image](https://github.com/Jawad-Nazari/Recipe-app/assets/66097371/7657c816-080d-48e8-934c-99117941aca8)

Reply:
Hi @VICTOR-CHUKWU ,

 Thanks for your advice. All the tests are passed now.
![image](https://github.com/Jawad-Nazari/Recipe-app/assets/115032263/9e6995df-c556-45ed-b311-4b27ff353e40)

Reply:
- Also, your tests are still not passing. kindly fix this.

![image](https://github.com/Jawad-Nazari/Recipe-app/assets/59397159/51a1e5d3-e15e-42bc-bc25-edb10cd6a924)